p2p/protocol/identify/id.go

```diff
@@ -41,6 +41,13 @@ const ID = "/ipfs/id/1.0.0"
 // 0.4.17 which asserted an exact version match.
 const LibP2PVersion = "ipfs/0.1.0"
 
+const (
+	// UDPNATDeviceTypeKey is the key with which we will persist a peer's UDP NAT Device Type to the peerstore.
+	UDPNATDeviceTypeKey = "UdpNATDeviceType"
+	// TCPNATDeviceTypeKey is the key with which we will persist a peer's TCP NAT Device Type to the peerstore.
+	TCPNATDeviceTypeKey = "TcpNATDeviceType"
```

Comment:
funky capitalization in the strings; it's `UDP` not `Udp` and same for `TCP`.

Reply:
Done.

Reply:
Done.